docs/content/en/docs/implementing/tasks/_index.md

```diff
@@ -0,0 +1,179 @@
+---
+title: Working with Keptn tasks
+description: Learn how to work with Keptn tasks
+weight: 90
+hidechildren: false # this flag hides all sub-pages in the sidebar-multicard.html
+---
+
+Keptn tasks are defined in a
+[KeptnTaskDefinition](../../yaml-crd-ref/taskdefinition.md/)
+resource.
+A task definition includes a function
+that defines the action taken by that task.
+It can be configured in one of three different ways:
+
+- inline
+- referring to an HTTP script
+- referring to another `KeptnTaskDefinition`
```

Comment:
Also, we support storing the function to a ConfigMap and reference it

Reply:
added